### PR TITLE
Tic tac toe edits josh

### PR DESCRIPTION
**Pull request by jmrodri3**
Edits for tic tac toe curriculum. Followed the suggested edits from the teams curriculum wiki.

### CODE REVIEW

File: content/english/java-tictactoe/activity-2.md [hunk 1]
```diff
@@ -73,62 +80,66 @@ Exception in thread "main" java.util.InputMismatchException      <------ tells y
...
+## 5. Handling Invalid Inputs with Scanner
+The Scanner class provides methods which check for valid or invalid inputs. For example, the method
 ```java
-String s1 = "hi";
-boolean b1 = s1.equals("hi"); // this is true
-boolean b2 = s1.equals("HI"); // this is false
+public boolean hasNextInt();
 ```
-{{% /notice %}}
+will check if the value of the user's input is an integer and returns true or false. This method can be used to ensure that the user's input is valid.
```
Comment (ozhang22) [inline content/english/java-tictactoe/activity-2.md:88]:
```suggestion
will check if the value of the user's input is an integer and returns `true` or `false`. This method can be used to ensure that the user's input is valid. 
```

File: content/english/java-tictactoe/placeholder.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Read and Process Input
```
Comment (ozhang22) [inline content/english/java-tictactoe/placeholder.md:1]: Please add metadata for this page, and set draft: true

**Review by ozhang22 — APPROVED** [review]

File: content/english/java-tictactoe/placeholder.md (hunk 2 above)
Comment (jmrodri3) [reply content/english/java-tictactoe/placeholder.md:1]: This page is a placeholder for when the Java - Advanced curriculum is created.  Should I still add the metadata ?

Comment (ozhang22) [reply content/english/java-tictactoe/placeholder.md:1]: OK, never mind then. Just make sure that if you try to go to the relative URL /java-tictactoe/placeholder in the website, that you don't end up seeing this page...

**Pull request merged**